jobs/JGDAS_GLOBAL_OCEAN_ANALYSIS_POST

```diff
@@ -5,7 +5,7 @@ source "${HOMEgfs}/ush/preamble.sh"
 ##############################################
 # make temp directory
 ##############################################
-export DATA=${DATA:-${DATAROOT}/ocnanal_${CDATE}} # TODO (G): Switch to {cyc} when the downstream code is ready
+export DATA=${DATA:-${DATAROOT}/ocnanal_${PDY}${cyc}} 
```

Comment:
I think @aerorahul wants instead this to be `${RUN}ocnanal_${cyc}` per a conversation he and I had yesterday afternoon

Reply:
Yes.  Thanks @CoryMartin-NOAA.
`${PDY}` is not available at this time, so the issue is correct on that, that `${CDATE}` should become `${cyc}`.

I also realized yesterday, that our run directories will get clobbered when `gdas` and `gfs` cycles start.  To distinguish between those, it is best to prepend `ocnanal` with a `${RUN}`.

`${RUN}` is set upstream of the job in the ecflow/rocoto job-card.  EMC internally translates that to `${CDUMP}`, which is also not available at this point in the j-job.